build/cmake/DefineOptions.cmake

```diff
@@ -164,31 +164,36 @@ message(STATUS "Thrift package version:                       ${PACKAGE_VERSION}
 message(STATUS "Build configuration Summary")
 message(STATUS "  Build Thrift compiler:                      ${BUILD_COMPILER}")
 message(STATUS "  Build compiler plugin support:              ${WITH_PLUGIN}")
+if (WITH_PLUGIN_COMPILER_TOO_OLD)
+  MESSAGE_DEP(WITH_PLUGIN "Disabled due to older compiler")
```

Comment:
If I do this and leave the MESSAGE_DEP(WITH_PLUGIN below, you get two messages.  It should be fine the way it is?